scripts/code_formatter.sh

```diff
@@ -0,0 +1,6 @@
+#Script for auto formatting python files
+
+pip install yapf
```

Comment:
instead of having pip install commands in the `.sh` file, we should instead add it to our `requirements.txt`